2008-10-16  Michael Tautschnig  <mt@debian.org>

	* lib/setup-storage/Commands.pm: fixed dependency error in vgcreate commands
		(closes: #502462)
Index: trunk/lib/setup-storage/Commands.pm
===================================================================
--- trunk.orig/lib/setup-storage/Commands.pm
+++ trunk/lib/setup-storage/Commands.pm	
@@ -292,9 +292,9 @@
       "pv_done_$_" ) foreach (@devices);
     # create the volume group
     my $pre_dev = "";
-    $pre_dev .= ",pv_done_$_" foreach (@devices);
+    $pre_dev .= ",exist_$_,pv_done_$_" foreach (@devices);
     $pre_dev =~ s/^,//;
-    &FAI::push_command( "vgcreate $vg " . join (" ", @devices), "exist_$pre_dev",
+    &FAI::push_command( "vgcreate $vg " . join (" ", @devices), "$pre_dev",
       "vg_created_$vg" );
     # we are done
     return;
